rpcs3/CMakeLists.txt

```diff
@@ -58,6 +58,10 @@ else()
 	set(ADDITIONAL_LIBS "")
 endif()
 
+if(UNIX)
```

Comment:
OS X support is kinda "abandoned" right now due to Apple's policy that lead to no availability of even remotely recent open graphics API on their system (the most recent OGL is 4.1, and it looks like there is no hope for newer versions/Vulkan since Apple is pushing for it's proprietary Metal API). So it was building but not useful without working renderer for quite some time.